content/api/index.apib

```diff
@@ -107,7 +107,7 @@ Features that are limited to some accounts will be marked appropriately througho
 To prevent abuse, our servers enforce request rate limiting, which may trigger responses with HTTP status code 429.
 The following endpoints are rate limited:
 
-- `/api/v1/message-events`
+- `/api/v1/events/*`
```

Comment:
Was removing `message-events` on purpose? should we keep both events and message-events?  there are plenty of other endpoints that are rate limited as well

Reply:
I thought I'd change the example service to one that wasn't being deprecated 😄